packages/api-explorer/src/index.jsx

```diff
@@ -184,7 +184,7 @@ class ApiExplorer extends React.Component {
 
     const defaultOpenDoc = this.props.defaultOpenDoc ? this.props.defaultOpenDoc : '0'
     const defaultOpen = this.props.defaultOpen ? [defaultOpenDoc] : null
-    const localizedMessages = messages[this.props.i18n.locale] || messages[this.props.i18n.defaultLocale]
+    const localizedMessages = messages[this.props.i18n.locale] || messages[this.props.i18n.defaultLocale] || messages.en
```

Comment:
Default locale should already be en. Why is it added the last or statement?

Reply:
It wasn't, it happens that passing `{ locale: 'en}` as prop overrides the default prop
```
i18n: {
    locale: 'en',
    defaultLocale: 'en',
  }
```

Reply:
It override the `defaultLocale` prop? Here the code try to access using `locale`, `defaultLocale` if locale is not defined. But when one of those are not in i18n object?

Reply:
I was looking that if the `API-Portal website` use this component and pass as prop `i18n = { locale: 'en'}`  then the `defaultLocale` here is `undefined`. That's why I was saying that the `defaultLocale` prop is overridden

Reply:
So the issue in `API-Portal website` is a not correct use of this library, I suppose. Is it possible to add the default locale in that website?

A fix for this library could be set the locale and defaultLocale as required, what do you think about it?

Reply:
yes, for sure I can add the default locale in the `API-Portal website`, also setting locale and defaultLocale as required should solve the problem!